mgmt/rest/v2/config.go

```diff
@@ -105,7 +139,7 @@ func (s *apiV2) deletePluginConfigItem(w http.ResponseWriter, r *http.Request, p
 		res = s.configManager.DeletePluginConfigDataNodeField(typ, name, iver, src...)
 	}
 
-	item := &PluginConfigItem{ConfigDataNode: res}
+	item := &PluginConfigItem{res}
```

Comment:
It's unlikely IMO.  Like mentioned above the response struct is to effectively support generating the client.